File: JavaScript_12.18/1272 - Mensagem Oculta.js
/**
 * @author Josélio de S. C. Júnior <joseliojrx25@gmail.com>
 * @copyright Josélio de S. C. Júnior 2021
 * @submission #23512907
 */
const p = require('fs').readFileSync('/dev/stdin', 'utf8'),
l = p.split('\n'),
t = l[0];

for (let i = 1; i <= t; i++) {
    console.log(l[i]
        .replace(/\s+/g, ' ')
        .split(' ')
        .map(x => x[0])
        .join(''));
}

File: Kotlin_1.2.10/1272 - Mensagem Oculta.kt
/**
 * @author Josélio de S. C. Júnior <joseliojrx25@gmail.com>
 * @copyright Josélio de S. C. Júnior 2021
 * @submission #23515189
 */
fun main(args: Array<String>) {
    val t = readLine()?.toInt() ?: 0

    for (i in 1..t) {
        println(
            readLine()!!.replace("\\s+".toRegex(), " ")
            .replace("^\\s|\\s$".toRegex(), "")
            .split(" ")
            .map{x -> x[0]}
            .joinToString("")
        )
    }
}